src/ui/components/SecondaryToolbox/redux-devtools/ReduxDevToolsListItem.module.css

```diff
@@ -1,6 +1,7 @@
 .ListItem {
   line-height: 1.5rem;
   padding: 0 1ch;
+  white-space: nowrap;
```

Comment:
This is not technically related but I noticed it while I was in there.
### Before
![Screenshot 2024-06-14 at 2 30 47 PM](https://github.com/replayio/devtools/assets/29597/5d61f3c0-17ae-47b1-8871-1fdfbed1bd32)
### After
![Screenshot 2024-06-14 at 2 31 04 PM](https://github.com/replayio/devtools/assets/29597/d561b922-a49d-4b70-a169-258149d9bc51)